kommunicate/src/main/java/io/kommunicate/KmConversationHelper.java

```diff
@@ -694,10 +694,10 @@ private static KmCallback getCallbackWithAppSettingsToCreateConversation(final b
         return new KmCallback() {
             @Override
             public void onSuccess(Object message) {
-                KmAppSettingModel.KmResponse kmAppSettings = (KmAppSettingModel.KmResponse) message;
+                KmAppSettingModel kmAppSettings = (KmAppSettingModel) message;
                 if (kmAppSettings != null) {
                     List<String> agents = new ArrayList<>();
-                    agents.add(kmAppSettings.getAgentId());
+                    agents.add(kmAppSettings.getResponse().getAgentId());
```

Comment:
@amntoppo How are we handling error here, will it create any issue if response from server is not proper or call failed?

Reply:
@adarshmishra it will show "Unable to create conversation" for this scenario. For Pseudo name, it will create a user without the pseudoname.